### PR TITLE
feat: add aggregation example

### PR DESCRIPTION
**Pull request by rorsini**
Resolves FT-e3301f68-820a-481a-bfe2-f1935b020508

- [ ] I have added automatic tests where applicable.
- [ ] The PR contains a description of what has been changed.
- [ ] The description contains manual test instructions.
- [ ] The PR contains updates to the release notes.
- [ ] I have verified that the documentation is still up to date.

## Changes

Rendered docs:

Screenshot 2023-06-30 at 1 41 01 PM

Screenshot 2023-06-30 at 1 41 12 PM

### CODE REVIEW

File: doc/example/aggregations.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Aggregation functions also require “group by” clause that specifies
+the attributes that will be used to distinguish how the aggregate results are
+grouped, such as::
+
+    select count(id) from Task group by status_id
+
+.. note::
+
+  Queries that include aggregations have to use the operations syntax (see:
+  https://help.ftrack.com/en/articles/1040498-operations) and be passed to the
```
Comment (bjornrydahl) [inline doc/example/aggregations.rst:53]: old docs.

File: doc/example/aggregations.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
+..
+    :copyright: Copyright (c) 2015 ftrack
```
Comment (bjornrydahl) [inline doc/example/aggregations.rst:2]: 2023

**Review by octavian-ionescu — APPROVED** [review]
lgtm and examples run fine.

File: doc/example/group_by.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        strict_api=True
+    )
+
+Aggregation functions also require a “group by” clause that specifies how the
+results are aggregated::
+
+    select count(id) from Task group by status_id
+
+Queries that include group by clauses must be executed using the
+:func:`session.call()` method versus :func:`session.query()`.
```
Comment (octavian-ionescu) [inline doc/example/group_by.rst:50]: Do we want to also include the new version of the help docs [https://help.ftrack-studio.backlight.co/hc/en-us/articles/13129860019863-Operations](url) link as well?

**Pull request merged**